File: VB/WindowsFormsApp13/Form1.vb
Imports DevExpress.XtraEditors
Imports System

Namespace WindowsFormsApp13
    Partial Public Class Form1
        Inherits XtraForm

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles Me.Load
            Me.reportDesigner1.OpenReport("CityOrders.repx")
        End Sub
    End Class
End Namespace

File: CS/WindowsFormsApp13/Form1.cs
using DevExpress.XtraEditors;
using System;

namespace WindowsFormsApp13 {
    public partial class Form1 : XtraForm {
        public Form1() {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e) {
            this.reportDesigner1.OpenReport("CityOrders.repx");
        }
    }
}
